airflow/config_templates/default_airflow.cfg

```diff
@@ -610,6 +610,10 @@ worker_pods_creation_batch_size = 1
 # The Kubernetes namespace where airflow workers should be created. Defaults to `default`
 namespace = default
 
+# If True (default False), any worker pods brought up will have the same environment variables as the container
+# that created them. These can be overwritten by kubernetes secrets and environment variables.
+mount_current_environment = False
```

Comment:
I've definitely considered the possibility that it could cause problems, but I haven't been able to come up with examples of when this could happen. The main problem that this is addressing is that my scheduler and worker pods have the same configuration, and manually specifying environment variables in my deployment file and then again in the `airflow.cfg` is error prone. An alternative might be to specify a ConfigMap that the worker pods `envFrom`.

Reply:
What you're saying makes sense, if more people agree that this won't cause issues I'd be fine with it, I like the idea of sharing a Configmap instead maybe you could change the configuration to be something like
```
# ---
# airflow-configmap.yaml:
# apiVersion: v1
# kind: ConfigMap
# metadata:
#   name: airflow-configmap
# data:
#   worker.env: |
#       ...
worker-env-from-configmap = airflow-configmap
```